app/views/admin/enterprises/form/_white_label.html.haml

```diff
@@ -1,6 +1,23 @@
+- @object ||= enterprise
+
 .row
   .three.columns.alpha
     = f.label :preferred_hide_ofn_navigation, t('.hide_ofn_navigation')
   .three.columns
     = f.check_box :preferred_hide_ofn_navigation
     
+.row{id: "white_label_logo"}
+  .three.columns.alpha
+    = f.label :white_label_logo, t('.upload_logo')
+  .three.columns
+    = image_tag @object.white_label_logo_url if @object.white_label_logo.variable?
+    = f.file_field :white_label_logo, accept: "image/*"
+    - if @object.white_label_logo.variable?
```

Comment:
👀 First time I've seen `variable?`. I can see it is used to determine the image's presence elsewhere in the codebase, but I can't find any info about this method online!

Reply:
https://api.rubyonrails.org/v6.1.7.3/classes/ActiveStorage/Blob/Representable.html#method-i-variable-3F

Reply:
Some images can be resized, others can't.

Reply:
So we only display images that can be resized (or otherwise reprocessed)? Seems like an indirect way of checking for a valid image 🤷

Reply:
Changed to `.present?`

Reply:
Good, it wasn't the right check here before. The `variable?` is important before trying to display an image as a certain variant because that will just raise an error if it's not variable.